### ISSUE TITLE
scheduling based on day

### ISSUE THREAD

**Issue by liam8888999** [ErsatzTV/ErsatzTV#904]
e.g. different schedule on Monday, Tuesday etc

**Comment by scoobshub** [comment]
yes, please, this would be very cool! (also thinking about different programming on weekends)

**Comment by Gryphon324** [comment]
I'd love this feature.

**Comment by jasongdove** [comment]
Planning for a new block scheduling feature (including day templates) has started. If anyone has specific scheduling needs please hop in the discord `#block-scheduling` channel and let me know.

**Comment by embolon** [comment]
This is already implemented, right? Probably should close?

**Comment by jasongdove** [comment]
Classic Schedules and Block Schedules both support scheduling based on day.
